confapp/pubspec.yaml

```diff
@@ -12,9 +12,7 @@ dependencies:
     sdk: flutter
   rohd: ^0.6.1
   rohd_hcl:
-    git: 
-      url: https://github.com/intel/rohd-hcl
-      ref: main
+    path: ../
```

Comment:
while this is nice for development and hosting on rohd-hcl, i realized this makes it difficult for people to use the confapp for configuring their own designs by depending on this confapp package.  is there a reason we need to make it point to `../`?